lib/chef/resource/macosx_service.rb

```diff
@@ -21,37 +21,19 @@
 class Chef
   class Resource
     class MacosxService < Chef::Resource::Service
-
+      resource_name :macosx_service
       provides :macosx_service, os: "darwin"
```

Comment:
delete the above line as well, the next line for `:service` is clearly correct though.